lib/postal-address.ts

```diff
@@ -133,7 +133,7 @@ class PostalAddress implements PostalAddressInterface {
     return null
   }
 
-  private output(overrideFormat: string): string[][] | string | null {
+  public output(overrideFormat: string): string[][] | string | null {
```

Comment:
Can you also add this to the [declaration file][declaration-file], since it's also a public method?

<!-- References -->

[declaration-file]: https://github.com/joaocarmo/i18n-postal-address/blob/master/lib/types/postal-address.ts

Reply:
Done!